docs/getting-started/your-first-set-of-documentation.rst

```diff
@@ -195,18 +197,21 @@ For example::
 
     $ phpdoc -d ./src -t ./docs/api
 
-What the above example does is scan all files in the ``src`` directory and its subdirectories, perform an analysis and
-generate a website containing the documentation in the folder ``docs/api``. If you want you can even omit the ``-t``
-option, in which case the output will be written to a subfolder called ``output``.
+What the above example does, is scan all files in the ``src`` directory and its subdirectories, perform
+an analysis and generate a website containing the documentation in the folder ``docs/api``. If you want,
+you can omit the ``-t`` option, in which case the output will be written to a subfolder called ``output``.
 
-.. hint::
+Read more
+~~~~~~~~~
+
+PhpDocumentor features several templates_ with which you can change the appearance of your documentation.
```

Comment:
Do we still have "several" templates?

Reply:
We do, however we are dropping most of them :-)